content/05.discussion_and_conclusion.md

```diff
@@ -0,0 +1,27 @@
+## Discussion and Conclusions
+
+Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
+Using manual curation alone becomes impractical as the rate of publications continuously increases.
+Data programming is a paradigm that uses label functions to speed up the annotation process and can be used to solve this problem.
+However, creating useful label functions is an obstacle to this paradigm, which takes considerable time.
+We tested the feasibility of re-using label functions to reduce the number of label functions required for strong prediction performance.
+
+Through our sampling experiment, we found that adding edge-specific label functions increases performance for the generative model (Figure {@fig:auroc_gen_model_test_set}).
+We found that label functions designed from relatively related edge types can increase performance (Gene-interacts0Gene (GiG) label functions predicting the Compound-binds-Gene (CbG) edge and vice versa), while the Disease-associates-Gene (DaG) edge type remained agnostic to label function sources (Figure {@fig:auroc_gen_model_test_set} and Supplemental Figure {@fig:aupr_gen_model_test_set}).
+Furthermore, we found that using all label functions at once generally hurts performance with the exception being the DaG edge type (Supplemental Figures {@fig:auroc_grabbag_gen_model_test_set} and {@fig:aupr_grabbag_gen_model_test_set}).
+One possibility for this observation is that DaG is a broadly defined edge type.
+For example, DaG may contain many concepts related to other edge types, such as Disease (up/down) regulating a Gene, which makes it more agnostic to label function sources (examples highlighted in our [annotated sentences](https://github.com/greenelab/text_mined_hetnet_manuscript/tree/master/supplementary_materials/annotated_sentences)).  
+
+Regarding the discriminative model, adding edge-specific label function improved performance for two out of the four edge types (Compound-treats-Disease (CtD) and  DaG) (Figure {@fig:auroc_discriminative_model_performance} and Supplemental Figure {@fig:aupr_discriminative_model_performance}). 
```

Comment:
Could this entire paragraph be summarized as: "In this context, the discriminative model did not have a clear positive or negative effect on performance."

I am worried this is so detailed that it's restating the results.